packages/front-end/src/app/login-page.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './add-person.module.scss';
+export function Login(): JSX.Element {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+  
+    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+      try {
+        const authToken = localStorage.getItem('authToken');
+        const response = await fetch('/login', {
+          method: 'POST',
+          headers: { 
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${authToken}`,
+          },
+          body: JSON.stringify({ email, password}),
+        });
+        const data = await response.json();
+        console.log(data);
```

Comment:
data should be in the format { token: string }, so the token should be persisted here (local storage or react context)